src/city/labor.c

```diff
@@ -47,7 +47,7 @@ static const int CATEGORY_FOR_BUILDING_TYPE[] = {
     0, 0, 8, 2, 5, 0, 0, 0, 6, 0, // 150
     6, -1, -1, -1, -1, -1, -1, -1, -1, -1, // 160
     -1, -1, -1, 5, -1, -1, 0, -1, -1, -1, // 170
-    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, // 180
+    -1, -1, -1, -1, 0, 0, -1 -1, -1, -1, // 180
```

Comment:
An innocent comma got removed here, I think.

Reply:
Good catch! That array is a mess...